db/migrate/20150121202952_add_processed_at_to_order_import.rb

```diff
@@ -0,0 +1,13 @@
+class AddProcessedAtToOrderImport < ActiveRecord::Migration
+  def up
+    add_column :order_imports, :processed_at, :timestamp, after: :created_by
+
+    OrderImport.where("error_file_id IS NOT NULL").each do |order_import|
```

Comment:
I suppose it could just be `OrderImport.all`, assuming (probably safely) that all existing `OrderImport`s would have been processed when this migration runs.